zerver/openapi/zulip.yaml

```diff
@@ -8361,6 +8361,25 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+    delete:
+      operationId: delete-custom-emoji
+      summary: Delete custom emoji
+      tags: ["server_and_organizations"]
+      description: |
+        This endpoint is used to delete a custom emoji from the user's
+        organization.
```

Comment:
We need a `**Changes**` entry here, with an API feature level bump and API changelog entry -- this is true for any API change, no matter how small.

https://zulip.readthedocs.io/en/latest/tutorials/new-feature-tutorial.html#update-documentation has details on how to do this correctly..

Also, the wording for the description feels a bit off in style compared with other endpoints; can you do an edit pass?

Reply:
What would you like in the description? I just followed `upload-custom-emoji` description style wording for it.

Reply:
I think it would be better to cut the first part and start with the verb, "Deactivate a custom emoji". I think it would make sense to make a similar change to the `api/upload-custom-emoji` documentation as well, but as a separate pull request.

Also, you'll want to use relative links for the help center in the API documentation (`/help/...`). And there's no linter rule, but in general it's good to keep the line breaks in this file similar to other sections.